protocol/streaming/grpc/grpc_streaming_manager.go

```diff
@@ -0,0 +1,38 @@
+package grpc
+
+import (
+	"github.com/dydxprotocol/v4-chain/protocol/streaming/grpc/types"
+	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+)
+
+var _ types.GrpcStreamingManager = (*GrpcStreamingManagerImpl)(nil)
+
+// GrpcStreamingManagerImpl is an implementation for managing gRPC streaming subscriptions.
+type GrpcStreamingManagerImpl struct {
+}
+
+func NewGrpcStreamingManager() *GrpcStreamingManagerImpl {
+	return &GrpcStreamingManagerImpl{}
+}
+
+func (sm *GrpcStreamingManagerImpl) Enabled() bool {
+	return true
+}
+
+// Subscribe subscribes to the orderbook updates stream.
```

Comment:
maybe add a comment about lifecycles of the req, srv, finished variables. i think there's a stream and a chan going on so not too sure when the finished chan returns a bool.